analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SonarAnalyzer.ShimLayer.CodeGeneration.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.6.0" />
```

Comment:
The actual change - for source generators => build-time only change.

Reply:
The link in this comment does not work for me. Could you please replace it? I'd like to see the full context.

Reply:
There's no  link. It's GitHub UI making things up to confuse you